panel/config/addplugindialog.cpp

```diff
@@ -81,6 +81,7 @@ AddPluginDialog::AddPluginDialog(QWidget *parent):
 AddPluginDialog::~AddPluginDialog()
 {
     delete ui;
+    ui = nullptr;
```

Comment:
No need for this...as this is a destructor and so all the members get out of scope right after this call...

Reply:
If you insist in this 'delete x; x = nullptr;` pattern, I would prefer to go more deeply and use smart pointers.